build/kube-bench/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM golang:1.14.0 AS build
+
+ARG VERSION
```

Comment:
Missing `ARG TARGETPLATFORM`

Reply:
Yeah, I noticed that with the missing substitution in the output, also had a type in the git repo.